### PR TITLE
fix: Support crust calc

### PR DESCRIPTION
**Pull request by yashirooooo**

### CODE REVIEW

**Comment by TarikGul** [comment]
Hey @yashirooooo, thanks for the PR. I will have a closer look at it later today when things free up. But for now from a quick glance I can see a few things missing. Here's a quick guide to get you started.

https://github.com/paritytech/substrate-api-sidecar/blob/master/guides/CHAIN_FEE_INTEGRATION.md

**Comment by yashirooooo** [comment]
> 

Thanks, I'll do it asap

File: .env.local [hunk 1]
```diff
@@ -3,4 +3,4 @@
 # For more information on how to use .env files and environment variables
 # consult the Configuration section in the README.
 
-SAS_SUBSTRATE_WS_URL=ws://127.0.0.1:9944
+SAS_SUBSTRATE_WS_URL=ws://127.0.0.1:9944
```
Comment (TarikGul) [inline .env.local:6]: Needs end of file line here.

File: calc/pkg/package.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+    },
+    "files": [
+      "calc_bg.wasm",
+      "calc.js",
+      "calc.d.ts"
+    ],
+    "main": "calc.js",
+    "homepage": "https://github.com/paritytech/substrate-api-sidecar/tree/master/calc#readme",
+    "types": "calc.d.ts"
+  }
```
Comment (TarikGul) [inline calc/pkg/package.json:21]: end of line here too, package was auto updated via wasm build?

**Review by TarikGul — COMMENTED** [review]
Overall looks good, you might need to pull from master as another chains fee calc was also added, and might cause merge conflicts.

File: .env.local (hunk 1 above)
Comment (yashirooooo) [reply .env.local]: Thanks, i will fix this

File: src/chains-config/crustControllers.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { ControllerConfig } from '../types/chains-config';
+import { initLRUCache } from './cache/lruCache';
+import { getBlockWeight } from './metadata-consts';
+
+/**
+ * Controllers for Acala collator
```
Comment (niklasad1) [inline src/chains-config/crustControllers.ts:6]:
```suggestion
 * Controllers for crust collator
```

**Review by TarikGul — APPROVED** [review]
👍

**Review by niklasad1 — APPROVED** [review]

**Pull request merged**